GeoMesa_Feature_List.md

```diff
@@ -0,0 +1,100 @@
+    - Supported file formats: "tif", "tiff", "geotiff", "dt0", "dt1", "dt2"
+
+
+- geomesa-convert (tools for converting various serialization formats to `SimpleFeature`s for ingestion - conversion mechanisms are specified by way of configuration files)
+  - delimited text (usually CSV/TSV)
+    - Pointer: [DelimitedTextConverter.scala](https://github.com/locationtech/geomesa/blob/b7056fae4988ef524913bf3dc33d9ff2a3476b09/geomesa-convert/geomesa-convert-text/src/main/scala/org/locationtech/geomesa/convert/text/DelimitedTextConverter.scala)
+    - Currently supported formats: "CSV" | "DEFAULT", "EXCEL", "MYSQL", "TDF" | "TSV" | "TAB", "RFC4180", "QUOTED", "QUOTE_ESCAPE", "QUOTED_WITH_QUOTE_ESCAPE". $1 through $n for n values per line ($0 refers to the entire line).
+  - fixed width
+    - Pointer: [FixedWidthConverters.scala](https://github.com/locationtech/geomesa/blob/b7056fae4988ef524913bf3dc33d9ff2a3476b09/geomesa-convert/geomesa-convert-fixedwidth/src/main/scala/org/locationtech/geomesa/convert/fixedwidth/FixedWidthConverters.scala)
+  - avro
+    - Pointer: [geomesa-convert-avro](https://github.com/locationtech/geomesa/tree/b7056fae4988ef524913bf3dc33d9ff2a3476b09/geomesa-convert/geomesa-convert-avro/src/main/scala/org/locationtech/geomesa/convert/avro)
+  - json
+    - Pointer: [geomesa-convert-json](https://github.com/locationtech/geomesa/tree/b7056fae4988ef524913bf3dc33d9ff2a3476b09/geomesa-convert/geomesa-convert-json/src/main/scala/org/locationtech/geomesa/convert/json)
+  - xml
+    - Pointer: [geomesa-convert-xml](https://github.com/locationtech/geomesa/tree/b7056fae4988ef524913bf3dc33d9ff2a3476b09/geomesa-convert/geomesa-convert-xml/src/main/scala/org/locationtech/geomesa/convert/xml)
+- geomesa-stream (support for streaming input)
+  - A datastore which listens for updates from a [source which meets certain conditions](https://github.com/locationtech/geomesa/blob/7c295d68bad92291e4260273134219dd0f938faf/geomesa-stream/geomesa-stream-api/src/main/scala/org/locationtech/geomesa/stream/SimpleFeatureStreamSource.scala)
+    - Pointer: [StreamDataStore.scala](https://github.com/locationtech/geomesa/blob/master/geomesa-stream/geomesa-stream-datastore/src/main/scala/org/locationtech/geomesa/stream/datastore/StreamDataStore.scala)
+    - A generic apache-camel based implementation](https://github.com/locationtech/geomesa/blob/b7056fae4988ef524913bf3dc33d9ff2a3476b09/geomesa-stream/geomesa-stream-generic/src/main/scala/org/locationtech/geomesa/stream/generic/GenericSimpleFeatureStreamSourceFactory.scala)
+  - Hooks for updating GeoServer on stream update
+    - Pointer: Implemented? [pomfile](https://github.com/locationtech/geomesa/tree/b7056fae4988ef524913bf3dc33d9ff2a3476b09/geomesa-gs-plugin/geomesa-stream-gs-plugin)
+
+## Data Processing
+
```

Comment:
another processing feature we support is data sampling - we haven't documented this well but an example can be seen here: https://github.com/locationtech/geomesa/blob/master/geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/index/Z3IdxStrategyTest.scala#L301